### PR TITLE
Show output of SUSEConnect

### PR DESCRIPTION
**Pull request by kravciak**
UNTESTED

### CODE REVIEW

File: tests/caasp/register_and_check.pm [hunk 1]
```diff
@@ -22,7 +22,7 @@ sub run {
     }
 
     # Check that registration was succeeded
-    assert_script_run 'SUSEConnect --status-text | grep -o "Status: ACTIVE"';
+    assert_script_run 'SUSEConnect --status-text | tee /dev/tty | grep -q "Status: ACTIVE"';
```
Comment (okurz) [inline tests/caasp/register_and_check.pm]: maybe https://superuser.com/a/914862/327890 is what you want:

```perl
assert_script_run 'SUSEConnect --status-text | tee /dev/tty | grep -z -q "Status: ACTIVE"';
```

**Comment by drpaneas** [comment]
I've just tested it works, at least it gives some feedback now.

http://skyrim.qam.suse.de/tests/10#step/register_and_check/3

LGTM for me

File: tests/caasp/register_and_check.pm (hunk 1 above)
Comment (kravciak) [reply tests/caasp/register_and_check.pm]: I welcome the idea, but I don't see difference in functionality when I try your example manually

Comment (kravciak) [reply tests/caasp/register_and_check.pm]: Please explain, I merged so panos can analyze failures meanwhile..

**Pull request merged**